src/pages/ColabPage/ColabPage.tsx

```diff
@@ -31,9 +32,18 @@ export default function ColabPage() {
 
   const [isLoading, setIsLoading] = useState(true)
   const userContext = useContext(IdContext)
+
   if (!userContext) {
     throw new Error('IdContext not found')
   }
+  
+  const sideBarContext = useContext(SidebarContext)
+
+  if (!sideBarContext) {
```

Comment:
In the future, this should render some kind of error state instead of throwing an error